From: Sebastian Andrzej Siewior <sebastian@breakpoint.cc>
Date: Wed, 8 Jun 2022 20:45:32 +0200
Subject: Remove the provider section.

The provider section breaks libssl1.1 users. Remove it for now.

Link: https://bugs.debian.org/1011051
Signed-off-by: Sebastian Andrzej Siewior <sebastian@breakpoint.cc>
---
 apps/openssl.cnf | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/apps/openssl.cnf b/apps/openssl.cnf
index 03330e0120a2..215768bfe710 100644
--- a/apps/openssl.cnf
+++ b/apps/openssl.cnf
@@ -51,11 +51,11 @@ tsa_policy3 = 1.2.3.4.5.7
 # .include fipsmodule.cnf
 
 [openssl_init]
-providers = provider_sect
+# providers = provider_sect
 
 # List of providers to load
-[provider_sect]
-default = default_sect
+# [provider_sect]
+# default = default_sect
 # The fips section name should match the section name inside the
 # included fipsmodule.cnf.
 # fips = fips_sect
@@ -68,7 +68,7 @@ default = default_sect
 # becomes unavailable in openssl.  As a consequence applications depending on
 # OpenSSL may not work correctly which could lead to significant system
 # problems including inability to remotely access the system.
-[default_sect]
+# [default_sect]
 # activate = 1
 
 
